### PR TITLE
Comment out an unnecessary css import.

### PR DESCRIPTION
**Pull request by jasongrout**
See https://github.com/maartenbreddels/ipyvolume/issues/41#issuecomment-322870907

### CODE REVIEW

File: packages/controls/css/labvariables.css [hunk 1]
```diff
@@ -30,7 +30,8 @@ all of MD as it is not optimized for dense, information rich UIs.
 /*
  * Optional monospace font for input/output prompt.
  */
-@import url('https://fonts.googleapis.com/css?family=Roboto+Mono');
+ /* Commented out in ipywidgets since we don't need it. */
+/* @import url('https://fonts.googleapis.com/css?family=Roboto+Mono'); */
```
Comment (vidartf) [inline packages/controls/css/labvariables.css]: The comment seems strange to me ("in ipywidgets"). Also: Why is it commented out and not simply deleted?

Comment (vidartf) [inline packages/controls/css/labvariables.css]: Oh, sorry for the drive-by comments: I see now that is file is a simple copy from the JLab source.

Comment (jasongrout) [inline packages/controls/css/labvariables.css]: Yes - I wanted to give some indication for when the copy is updated what should happen.
